float32 temperature
float32 humidity
int32 gasses
int32 light
int32 rain
float32 pressure
float32 internalTemperature
int32 airParticlesDataAtmosferic1
int32 airParticlesDataAtmosferic2_5
int32 airParticlesDataAtmosferic10